frontend/public/components/dashboards-page/overview-dashboard/alerts-card.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { AlertsBody, AlertItem, filterAlerts } from '../../dashboard/alerts-card';
+import { DashboardItemProps, withDashboardResources } from '../with-dashboard-resources';
+import { RootState } from '../../../redux';
+import { featureReducerName, flagPending } from '../../../reducers/features';
+import { FLAGS } from '../../../const';
+import { ALERTS_KEY } from '../../../actions/dashboards';
+
+export const ALERT_MANAGER_BASE_PATH = window.SERVER_FLAGS.alertManagerBaseURL;
```

Comment:
why is this here ?

Reply:
This must have been left over from some stuff I had tried previously. It's been deleted now. Thanks for catching it!